src/Error/ErrorsManager.php

```diff
@@ -96,6 +96,8 @@ public function report(Error $error): void
         $this->errors[] = $error;
     }
 
+    // @TODO v4 move handling of WorkerException outside of ErrorsManager and break the execution flow when it happen @Wirone
+    // ref https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7777#discussion_r1591666901
```

Comment:
Not needed, as error handling has been [reworked](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7777#issuecomment-2109111690).